op_crates/crypto/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+[package]
+name = "deno_crypto"
+version = "0.1.0"
+edition = "2018"
+description = "Collection of WebCrypto APIs"
+authors = ["the Deno authors"]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/denoland/deno"
+
+[lib]
+path = "lib.rs"
+
+[dependencies]
+deno_core = { version = "0.63.0", path = "../../core" }
+rand = "0.7.3"
```

Comment:
It'd be great if we could remove `rand` dependency from `cli/` then - that would require to slightly change `ops/fs/` to use `tempfile` crate instead of `rand` - @littledivy could you look into that?

Reply:
`crypto.rs` is still dependent on rand - https://github.com/denoland/deno/pull/7956/files#diff-1edd7a8788f2b6027674aee860437562bcd17c1aa7cd0d0ef6c541548f0f2fe1R3-R4

Reply:
@littledivy you're right... I guess we could reexport `rand` from `deno_crypto` then